text/0061-defrag-memos.md

```diff
@@ -0,0 +1,68 @@
+- Feature Name: RTH Defragmentation Memos
+- Start Date: 2023-02-03
+- MCIP PR: [mobilecoinfoundation/mcips#0061](https://github.com/mobilecoinfoundation/mcips/pull/0061)
+
+# Summary
+[summary]: #summary
+
+This proposal introduces a new memo type to support [Recoverable Transaction History (RTH)](https://github.com/mobilecoinfoundation/mcips/pull/4). The new memo type will be used by clients to mark `TxOut`s created as part of account defragmentation. This can be used by client applications as a reliable way to exclude defragmentation transactions from appearing as regular transactions.
+
+# Motivation
+[motivation]: #motivation
+
+[RTH](https://github.com/mobilecoinfoundation/mcips/pull/4) introduced a standard method of utilizing [encrypted memos](https://github.com/mobilecoinfoundation/mcips/pull/3) for recording transaction history between two users. Not all transactions on the MobileCoin blockchain are between two different users or accounts, however. Defragmenation transactions are transactions that a user sends to themselves in order to consolidate small `TxOut`s into larger ones. This is necessary if a user needs to send a large transaction but cannot satisfy the amount using 16 `TxOut`s or less.
+
+From the client's perspective, these larger `TxOut`s created by defragmentation appear as `TxOut`s received through normal transactions. This would be confusing to users if displayed this way in an application. Currently, the only way to differentiate defragmentation transactions from regular ones is to use [RTH](https://github.com/mobilecoinfoundation/mcips/pull/4) and check if the sender and recipient are both the user's account. This isn't immediately clear though, especially to new developers who have not been introduced to concepts like fragmentation or [RTH](https://github.com/mobilecoinfoundation/mcips/pull/4). Thus, a new memo type is proposed to mark defragmentatino transactions.
```

Comment:
```suggestion
From the client's perspective, these larger `TxOut`s created by defragmentation appear as `TxOut`s received through normal transactions. This would be confusing to users if displayed this way in an application. Currently, the only way to differentiate defragmentation transactions from regular ones is to use [RTH](https://github.com/mobilecoinfoundation/mcips/pull/4) and check if the sender and recipient are both the user's account. This isn't immediately clear though, especially to new developers who have not been introduced to concepts like fragmentation or [RTH](https://github.com/mobilecoinfoundation/mcips/pull/4). Thus, a new memo type is proposed to mark defragmentation transactions.
```